pkg/ccl/backupccl/backup_test.go

```diff
@@ -5207,7 +5220,7 @@ func TestPointInTimeRecovery(t *testing.T) {
 		// incbackup.bank could be done here by the operator.
 
 		sqlDB.Exec(t, `DROP TABLE data.bank`)
-		sqlDB.Exec(t, `ALTER TABLE incbackup.bank RENAME TO data.bank`)
+		sqlDB.Exec(t, `CREATE TABLE data.bank AS SELECT * FROM incbackup.bank`)
```

Comment:
ditto DROP comment.

Reply:
Don't need to since we drop the database